src/main/java/org/jitsi/videobridge/shim/ChannelShim.java

```diff
@@ -248,12 +248,12 @@ public void describe(ColibriConferenceIQ.ChannelCommon commonIq)
     /**
      * Sets the list of sources signaled for this channel.
      */
-    public void setSources(@NotNull List<SourcePacketExtension> sources)
+    public void setSources(@NotNull List<SourcePacketExtension> sources, MediaType mediaType)
```

Comment:
The `ChannelShim` already has an associated `ContentShim`. I think it would be better to call `getMediaType` in the method instead of taking it as a param. Or is there a reason this won't work?

Reply:
I was looking for something like this and guess I just missed it.  Will change it.